diff --git a/entityqueue.module b/entityqueue.module
index 72f7b6e..b967993 100644
--- a/entityqueue.module
+++ b/entityqueue.module
@@ -1035,7 +1035,9 @@ function entityqueue_field_widget_form(&$form, &$form_state, $field, $instance,
         '#size' => $instance['widget']['settings']['size'],
         '#element_validate' => array('_entityreference_autocomplete_validate'),
         '#attributes' => array(
-          'id' => $table_id . '-add',
+          'class' => array(
+            $table_id . '-add',
+          ),
         ),
       ),
       'submit' => array(
@@ -1123,7 +1125,7 @@ function entityqueue_field_widget_ajax_callback($form, $form_state) {
 
 
   $add_id = drupal_clean_css_identifier('entityqueue-dragtable-' . $field_name . '-add');
-  $commands[] = ajax_command_invoke('#' . $add_id, 'val', array(''));
+  $commands[] = ajax_command_invoke('.' . $add_id, 'val', array(''));
   $settings = array(
     drupal_clean_css_identifier('entityqueue-dragtable-' . $field_name) => true,
   );
