diff --git a/js/search_autocomplete.admin.js b/js/search_autocomplete.admin.js
index 1aee8b8..077215e 100644
--- a/js/search_autocomplete.admin.js
+++ b/js/search_autocomplete.admin.js
@@ -48,7 +48,7 @@
     	
     	$("<ul id='sa_admin_menu'><div class='sa_title'>Search Aucomplete</div><li class='sa_add'>" + Drupal.t('add autocompletion') + "</li></ul>").appendTo( $('body') );
     	
-    	$(input_selector).live("mouseover", function (event) {
+    	$(input_selector).on("mouseover", function (event) {
     		var offset = $(this).offset();
     		
     		// display the context menu
@@ -64,23 +64,23 @@
     	});
     	
     	// hide the menu when out or used
-    	$(input_selector).live("click", function () {
+    	$(input_selector).on("click", function () {
     		$("#sa_admin_menu").hide();
     	});
-			$(input_selector).live("mouseout", function () {
+			$(input_selector).on("mouseout", function () {
 				$("#sa_admin_menu").hide();
 			});
 			
 			// hide the menu when out
-			$("#sa_admin_menu").live("mouseover", function(){
+			$("#sa_admin_menu").on("mouseover", function(){
 				$(this).show();
 			});
-			$("#sa_admin_menu").live("mouseout", function(){
+			$("#sa_admin_menu").on("mouseout", function(){
 				$(this).hide();
 			});
 			
 			// add a new autocompletion
-			$(".sa_add").live("click", function () {
+			$(".sa_add").on("click", function () {
 				window.location = 'index.php?q=admin/config/search/search_autocomplete/add&selector=' + encodeURI(selector.replace('#', '%23'));
     	});
 			
